### DISCUSSION TITLE
TImage32Panel not found in repository

### DISCUSSION THREAD

**Discussion by riaanjutte** [AngusJohnson/Image32#19] [General]
There are references on the authors website to a TImage32Panel component, but I can't find the unit containing it anywhere. Does anyone know where I can obtain it? Thanks!

**Comment by AngusJohnson** [comment]
https://github.com/AngusJohnson/Image32/blob/main/source/Img32.Panels.pas 😁

**Reply by riaanjutte** [reply]
I have no idea how I missed that, thanks! 😅
